### PR TITLE
Master codecov

### PR DESCRIPTION
**Pull request by dereuromark**
Resolves https://github.com/FriendsOfCake/travis/issues/71

Verified with https://travis-ci.org/dereuromark/cakephp-tools/jobs/107091620

Will better need a new version bump afterwards.

### CODE REVIEW

**Comment by davidyell** [comment]
:+1:

**Comment by dereuromark** [comment]
It all seems to work, but the result for 2.x is 0 coverage
https://travis-ci.org/dereuromark/cakephp-tools/jobs/107113007

Is there anything we need to change/fix in phpunit.xml for whitelist etc?
Or does anyone have an idea whats going on?

File: templates/.travis.yml [hunk 1]
```diff
@@ -12,17 +12,16 @@ env:
...
     - php: 5.4
       env:
         - DB=mysql
-        - CAKE_VERSION=2.4
-        - COVERALLS=1
+        - CAKE_VERSION=2.7
+        - CODECOVERAGE=1
     - php: 5.4
       env:
         - PHPCS=1
```
Comment (ADmad) [inline templates/.travis.yml]: Better to run phpcs build also using php 7.

**Comment by chinpei215** [comment]
@dereuromark Couldn't you keep the `COVERALLS` env? I tried to use codecov.io but my 2.x plugin got 0.00 coverage too. I think that the env can be coexisted with the `CODECOVERAGE` env.

**Comment by dereuromark** [comment]
You are right. You can make a PR.

**Comment by chinpei215** [comment]
OK, I'll do it on the next weekend.

**Pull request merged**